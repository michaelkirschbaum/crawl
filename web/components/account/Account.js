import React, { Component } from 'react'
import "./Account.css"

class Account extends Component {
  render() {
    return (
      <div className="Account">
      </div>
    );
  }
}

export default Account
