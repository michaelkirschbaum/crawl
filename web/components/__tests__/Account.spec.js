import React from 'react'
import Account from '../../components/account/Account'
import { shallow } from 'enzyme'

describe('Account', () => {
  it('should render correctly', () => {
    const component = shallow(<Account />)
    expect(component).toMatchSnapshot()
  })
})
