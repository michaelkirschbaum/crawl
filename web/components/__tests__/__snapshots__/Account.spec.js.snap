// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Account should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Account />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Settings
        </h1>,
        <label>
          Hi 
          
           
          
          !
        </label>,
        <div>
          <button
            onClick={[Function]}
          >
            Logout
          </button>
        </div>,
      ],
      "className": "Account",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Settings",
        },
        "ref": null,
        "rendered": "Settings",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Hi ",
            "",
            " ",
            "",
            "!",
          ],
        },
        "ref": null,
        "rendered": Array [
          "Hi ",
          "",
          " ",
          "",
          "!",
        ],
        "type": "label",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <button
            onClick={[Function]}
          >
            Logout
          </button>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Logout",
            "onClick": [Function],
          },
          "ref": null,
          "rendered": "Logout",
          "type": "button",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Settings
          </h1>,
          <label>
            Hi 
            
             
            
            !
          </label>,
          <div>
            <button
              onClick={[Function]}
            >
              Logout
            </button>
          </div>,
        ],
        "className": "Account",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Settings",
          },
          "ref": null,
          "rendered": "Settings",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Hi ",
              "",
              " ",
              "",
              "!",
            ],
          },
          "ref": null,
          "rendered": Array [
            "Hi ",
            "",
            " ",
            "",
            "!",
          ],
          "type": "label",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <button
              onClick={[Function]}
            >
              Logout
            </button>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Logout",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": "Logout",
            "type": "button",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
